app/views/partials/_navbar.html.haml

```diff
@@ -3,6 +3,7 @@
         =link_to image_tag('logo.png', {id: 'logo'}), root_path
 
 -if user_signed_in?
+    = link_to 'Users', users_path if current_user.editor?
```

Comment:
use the helper method `user_is_a_editor` here instead

Reply:
updated